python_modules/dagster/dagster/core/asset_defs/decorators.py

```diff
@@ -439,3 +441,17 @@ def build_asset_ins(
         ins_by_asset_key[asset_key] = (stringified_asset_key, In(cast(type, Nothing)))
 
     return ins_by_asset_key
+
+
+def _make_asset_keys(deps: Optional[Union[Set[AssetKey], Set[str]]]) -> Optional[Set[AssetKey]]:
+    """Convert all str items to AssetKey in the set."""
+    if deps is None:
+        return deps
+
+    deps_asset_keys = set()
```

Comment:
nit: would be more pythonic to do a set comprehension like:
```
deps_asset_keys = {AssetKey.from_user_string(dep) if isinstance(dep, str) else dep for dep in deps}
```

Reply:
Done.